### PR TITLE
Megahit

### PR DESCRIPTION
**Pull request by jhayer**
the build is working, need to tag and test the produced tar.gz

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,20 +2,24 @@ language: generic
 os:
     - linux
     - osx
+compiler:
+    - gcc
```
Comment (HadrienG) [inline .travis.yml]: is specifying the compiler still necessary since we force compiling against gcc on OSX anyway?

**Comment by HadrienG** [comment]
Only one small comment, looks good!

File: .travis.yml (hunk 1 above)
Comment (jhayer) [reply .travis.yml]: well, probably not needed indeed

**Comment by HadrienG** [comment]
The build errored for some reason, restarting now

**Comment by HadrienG** [comment]
merging and adding tag `0.2.2`, we'll see if the binaries work when Travis has done his thing 😊

**Pull request merged**